src/main.rs

```diff
@@ -69,28 +69,36 @@ fn main() {
 /// A Shop is a collection of 3 cards.
 struct Shop {
     // TOOD: Add the field to this struct
+    cards: [Card; 3],
 }
 
+use std::cmp;
+
 impl Shop {
     /// Get the price of the most expensive card in the shop
     fn most_expensive(&self) -> u32 {
-        todo!()
+        cmp::max(
+            self.cards[0].price,
+            cmp::max(self.cards[1].price, self.cards[2].price),
+        )
```

Comment:
This one would also do better to use a loop, so that it's easier to scale if the size of the array was larger :)